bot_config.py

```diff
@@ -1,7 +1,16 @@
 #!/usr/bin/python
 
 # Discord bot token
-bot_token = ''
+try:
+	bot_token = open('token', 'r').read()
+except:
+	bot_token = ''
+
+#cleans up bot commands in chat
+hide_cmds = True
```

Comment:
Won't these cause trouble with people spamming the bot? (Maybe tie this to some sort of log?)